[2025-03-31T08:24:57.618+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-31T08:24:57.639+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.first_task manual__2025-03-31T08:24:57.319362+00:00 [queued]>
[2025-03-31T08:24:57.650+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.first_task manual__2025-03-31T08:24:57.319362+00:00 [queued]>
[2025-03-31T08:24:57.650+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-31T08:24:57.669+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): first_task> on 2025-03-31 08:24:57.319362+00:00
[2025-03-31T08:24:57.676+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2222) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-31T08:24:57.678+0000] {standard_task_runner.py:72} INFO - Started process 2223 to run task
[2025-03-31T08:24:57.676+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'first_task', 'manual__2025-03-31T08:24:57.319362+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp8ysg1f7l']
[2025-03-31T08:24:57.679+0000] {standard_task_runner.py:105} INFO - Job 68: Subtask first_task
[2025-03-31T08:24:57.743+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v3.first_task manual__2025-03-31T08:24:57.319362+00:00 [running]> on host 9663692256f6
[2025-03-31T08:24:57.865+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-31T08:24:57.319362+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-31T08:24:57.319362+00:00'
[2025-03-31T08:24:57.868+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-31T08:24:57.886+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-31T08:24:57.887+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world, this is the first task!']
[2025-03-31T08:24:57.896+0000] {subprocess.py:99} INFO - Output:
[2025-03-31T08:24:57.898+0000] {subprocess.py:106} INFO - hello world, this is the first task!
[2025-03-31T08:24:57.898+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-31T08:24:57.922+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-31T08:24:57.923+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=first_task, run_id=manual__2025-03-31T08:24:57.319362+00:00, execution_date=20250331T082457, start_date=20250331T082457, end_date=20250331T082457
[2025-03-31T08:24:57.986+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-31T08:24:58.013+0000] {taskinstance.py:3901} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-03-31T08:24:58.015+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
